src/assets/wise5/components/common/feedbackRule/feedback-rule-help/feedback-rule-help.component.html

```diff
@@ -29,6 +29,33 @@ <h3 i18n>Terms</h3>
         Ex: <b>hasKIScore(3)</b> evaluates to true if the student received KI score 3.
       </td>
     </tr>
+    <tr>
+      <td class="term">accumulatedIdeaCountMoreThan(X)</td>
+      <td i18n>
+        Evaluates to true if more than X ideas were found in all (both past and current) of
+        student's responses.<br />
+        Ex: <b>accumulatedIdeaCountMoreThan(2)</b> evaluates to true if the student had more than 2
+        ideas in all of their responses.
+      </td>
+    </tr>
+    <tr>
+      <td class="term">accumulatedIdeaCountLessThan(X)</td>
+      <td i18n>
+        Evaluates to true if less than X ideas were found in all (both past and current) of
+        student's responses.<br />
+        Ex: <b>accumulatedIdeaCountLessThan(2)</b> evaluates to true if the student had less than 2
+        ideas in all of their responses.
+      </td>
+    </tr>
+    <tr>
+      <td class="term">accumulatedIdeaCountEquals(X)</td>
+      <td i18n>
+        Evaluates to true if exactly X ideas were found in all (both past and current) of student's
+        responses.<br />
+        Ex: <b>accumulatedIdeaCountEquals(2)</b> evaluates to true if the student had exactly 2
+        ideas in all of their responses.
+      </td>
+    </tr>
```

Comment:
This shows up in the Help section for Open Response and Dynamic Prompt even though it doesn't work in those places. Should we have it not show up in those places or add a footnote or something?